ImGuiApp/ImGuiApp.cs

```diff
@@ -147,17 +264,17 @@ public static void Start(AppConfig config)
 
 					inputContext = window.CreateInput(); // create an input context
 					controller = new ImGuiController.ImGuiController
-					(
-						gl,
-						view: window,
-						input: inputContext,
-						onConfigureIO: () =>
-						{
-							UpdateDpiScale();
-							InitFonts();
-							config.OnStart?.Invoke();
-						}
-					);
+			(
```

Comment:
Is this tabbing correct? It is a little difficult to assess in the diff.

Reply:
it is not

Reply:
fixed in 4fdfbf24057fe5e5471e6e75bd07a4ee927fe053